src/customJs/linkProperty.js

```diff
@@ -0,0 +1,38 @@
+const unlayer = window.unlayer;
+
+const customPropertyLabel = (label) => `
+            <div class="blockbuilder-widget-label mb-2">
+                <label class="blockbuilder-label-primary">
+                    <div class=""><span class="has-value same-value">${label}</span>
+                    </div>
+                </label>
+            </div>`;
+
+const linkPropertyWidget = {
+  render(value, updateValue, data) {
+    const label = data && data.label ? customPropertyLabel(data.label) : '';
+
+    return `
+            ${label}
+            <div class="href_field input-group">
+              <div class="input-group-prepend">
+                <span class="input-group-text">URL</span>
```

Comment:
El texto `URL` podrá sacarse de los literales nuestros o de los que trae Unlayer?

Reply:
En Unlayer no conozco forma de conocer los literales que ofrece, podría desde la traducción nuestra agregarlos

Reply:
Investigue en el ambiente que nos ofrece Unlayer y no vi nada respecto a los literales, decidí implementarlo en nuestra _INTL_

Reply:
I think it is the right resource: https://github.com/unlayer/translations/blob/31f3af740ed84c345df2f2fa1bbaa5cf22bedb0d/es.json#LL208C16-L208C16

Reply:
You can find the related documentation here: https://docs.unlayer.com/docs/localization

Reply:
Ohhh thanks, I didn't know that

Reply:
Updated, now we override the URL Unlayer's literal `editor.link.url`